packages/animation-plugin/babel.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function (api) {
+  api.cache(true);
+  return {
+    presets: ['babel-preset-expo'],
+    plugins: [
+      [
```

Comment:
Alias resolving should only work in development

Reply:
Done